### PR TITLE
Add Linear algebra operators to jax.lax Frontend "eig function"

### PR DESCRIPTION
**Pull request by DJALLIL05**

### CODE REVIEW

**Comment by DJALLIL05** [comment]
Do I have to get all the checks to pass ?

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @DJALLIL05 

Would you indicate the issue of this function that you're implementing and also add the implementation along with the unit tests😊

Feel free to explore the contributing guide here https://lets-unify.ai/docs/ivy/overview/contributing.html

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
